.buildkite/pipeline.yml

```diff
@@ -1,10 +1,5 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
-env:
```

Comment:
we might need this, as this is propagated to the other steps of the dynamic pipeline, ea Filebeat/Metricbeat etc..